### ISSUE TITLE
Tensorflow version

### ISSUE THREAD

**Issue by karlengblom** [xbpeng/DeepMimic#164]
Hello! Thanks for providing such a cool repo. Just a question about the version of Tensorflow to be used. In the requirements.txt file it's >=1.3.0, in the readme/front page it states 1.13.1. Which version is the correct one?

**Comment by xbpeng** [comment]
The one in requirement.txt >= 1.3.0 should be good. I think 1.13.1 should be fine too.

**Comment by alan-man** [comment]
Warning : for me at least, did not work with tensorflow2

**Comment by xbpeng** [comment]
Yes, the project probably doesn't work with tensorflow2. I'll update requirements.txt.

**Comment by WilliamTambellini** [comment]
@xbpeng  would you consider/review a PR to upgrade to TF2 ?
